CONTRIBUTING.md

```diff
@@ -40,3 +40,9 @@ The labels I assign to issues are:
 * **unconfirmed**: More information is needed to be sure this bug is really a bug.
 * **unlikely**: An enhancement or other change that I consider lowest priority, too large or difficult, or not appropriate.
 * **wontfix**: A change that I definitely do not think should be made.
+
+Some additional labels might be used to specify the scope of issues:
+* **art**: Everything related to images, sprites, and graphics
+* **sound**: Everything to do with sound
+* **text - long**: Content that contains long blocks of text such as large missions and campaigns
```

Comment:
```suggestion
```
I don't really see any reason to add this to the contributing file, this specifically definitions of issue labels, there's not really any time we'd add text - long or text - short to an issue. Those labels are pretty much specifically for PRs. An issue would just use the more generic "content" label.